src/Converter/ConverterInterface.php

```diff
@@ -18,3 +18,13 @@ public function convert(ElementInterface $element);
      */
     public function getSupportedTags();
 }
+
+interface PreConverterInterface
```

Comment:
shouldn't this be in a separate file?

Reply:
If you are the kind of person who carelessly wastes files, sure.

Joking aside, @colinodell actually refactored some of this code after merging and it is in a separate file now.

Reply:
just wondered how this would play with composer :)

Reply:
I have no idea actually, it might not work this way. I know nothing about composer. This PR is the first time I have touched PHP in over a decade.